api/app/src/main/kotlin/packit/model/dto/SubmitRunInfo.kt

```diff
@@ -0,0 +1,8 @@
+package packit.model.dto
+
+data class SubmitRunInfo(
+    val name: String,
+    val branch: String,
+    val hash: String,
```

Comment:
```suggestion
    val commitHash: String,
```
Maybe a bit clearer, and since this is what it's called in the db...

Reply:
so the orderly runner just takes in hash not commit hash, we could do json property annotation and use commitHash in the kotlin code, idm but then the frontend will have to send `hash` as well anyway, im happy to change it and add the annotation though, what do you think?

Reply:
Ah, fair enough, not an issue then!